apache/yunikorn-web
# [YUNIKORN-2339] Remove Nodes Utilisation chart from Dashboard page (Web)

### What is this PR for?
We have implemented multi-type resource utilisation chart in [YUNIKORN-2325](https://issues.apache.org/jira/browse/YUNIKORN-2325).
Remove the old node utilisation chart from Dashboard page, as its functionality is duplicated.

1. Remove componenet app-node-utilization
2. Remove componenet app-bar-char
3. Remove all reference of  deprecated api "ws/v1/scheduler/node-utilization"



### What type of PR is it?

### Todos
NA

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2325

### How should this be tested?
- make lint
- make license-check
- yarn test:coverage
- yarn start:srv
- yarn start 


### Screenshots (if appropriate)
<img width="1464" alt="image" src="https://github.com/apache/yunikorn-web/assets/26764036/4e6ee740-8fd7-413d-9aa0-5de22f5d4f36">




### Questions:
NA
